### ISSUE TITLE
[Serverless V4] The "exclude" keyword doesn't work for esbuild

### ISSUE THREAD

**Issue by NicoHinderling** [serverless/serverless#12645]
### Issue description

I tried to use the "exclude" keyword like it's detailed in [the docs](https://www.serverless.com/framework/docs/providers/aws/guide/building), but when I do that I get the following error:

```
✖ Build failed with 1 error:
/Users/nicolashinderling/.serverless/releases/4.1.6/package/node_modules/esbuild/lib/main.js:253:12: ERROR: Invalid option in build() call: "exclude"
ServerlessError2: Build failed with 1 error:
/Users/nicolashinderling/.serverless/releases/4.1.6/package/node_modules/esbuild/lib/main.js:253:12: ERROR: Invalid option in build() call: "exclude"
    at Esbuild._build (file:///Users/nicolashinderling/.serverless/releases/4.1.6/package/dist/sf-core.js:1026:11533)
```

**Comment by NicoHinderling** [comment]
@Mmarzex guessing this is another simple fix, since it's just not in the main.js as an available option at the moment. Let me know if I'm missing something

**Comment by ArthurRaimann** [comment]
Same here.

**Comment by eahefnawy** [comment]
Hey folks, thanks a lot for reporting this. I fixed this in https://github.com/serverless/serverless/pull/12646 and will included in the next release 🚀

**Comment by Mmarzex** [comment]
This has been released with [4.1.7](https://github.com/serverless/serverless/releases/tag/v4.1.7)
